tests/__init__.py

```diff
@@ -9,3 +18,26 @@ def load_fixture(name):
     )
     with open(path, 'r') as f:
         return json.loads(f.read())
+
+
+def compare_item_with_dict(item: Item, data: dict):
```

Comment:
An alternative that could also be useful to check. Checking that `attr.asdict(item) == data`. The check can also be included in this function. I think both ways of checking are useful.

Reply:
`attr.asdict` would be perfect if we didn't need to verify child objects like `Breadcrumb` and `Offer`. I want to make sure they're being properly parsed as one of those `Item`s instead of being stored as a plain `dict`.

Reply:
Makes sense.

Reply:
Mmm, I'm thinking now that the method https://github.com/scrapinghub/autoextract-poet/blob/master/autoextract_poet/items.py#L10-L11 can be made generic by the use of  `attrs.fields_dict` https://www.attrs.org/en/stable/api.html#attr.fields_dict and then getting the type of the field for each attribute in the dict. If the type of the field is subclass of `Item`, then delegate the loading the field class. 

Also then this test can be make generic. So no need to modify it for new Items. What do you think?

Reply:
I see this is being addressed here: https://github.com/scrapinghub/autoextract-poet/pull/3/files :+1: